packages/os/kdump.service

```
@@ -0,0 +1,11 @@
+[Unit]
+Description=Create dump after kernel crash
```

`kdump.service` is a bit generic - at a quick glance I'd expect this to be the service that loads the crash kernel.

Maybe `capture-kernel-dump.service`, and align the name with the prairiedog subcommand.

```suggestion
[Unit]
Description=Capture dump after kernel crash
```